resource/bem/css.css

```diff
@@ -0,0 +1,270 @@
+    -moz-box-sizing: border-box;
+    -webkit-box-sizing: border-box;
+}
+
+
+/*page*/
+
+.page {
+    width: 900px;
+    margin: 0 auto;
+}
+
+.page__header {
+    height: 70px;
+    margin-bottom: 10px;
+    border: 1px solid red;
+}
+
+.page__content {
+    min-height: 100%;
+    border: 1px solid red;
+}
+
+.page__footer {
+    height: 70px;
+    margin-top: 10px;
+    border: 1px solid red;
+    text-align: center;
+}
+
+
+/*end page*/
+
+
+/*copyright*/
+
+.copyright {
+    line-height: 70px;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 10pt;
+    color: lightgray;
+}
+
+
+/*end copyright*/
+
+
+/*multi-column*/
+
+.multi-columns {
+    width: 100%;
+}
+
+.page__header__left {
+    width: 30%;
+    float: left;
+}
+
+.page__header__right {
+    width: 70%;
+    float: left;
+}
+
+.page__header__left>* {
+    float: left;
+}
+
+.page__header__right>* {
+    float: right;
+}
+
+
+/*end multi-column*/
+
+
+/*branch*/
+
+.branch {
+    margin-left: 30px;
+    margin-top: 10px;
+    padding: 5px;
+    display: inline-block;
+    border: 1px solid blue;
+}
+
+.branch__icon {
+    height: 40px;
+    width: 40px;
+}
+
+.branch__name {
+    font-family: Arial, Helvetica, sans-serif;
+    font-family: 14px;
+    vertical-align: top;
+}
+
+
+/*end branch*/
+
+
+/*main-nav*/
+
+.main-nav {
+    list-style-type: none;
+    text-align: center;
+    width: 100%;
+}
+
+.main-nav__item {
+    float: left;
+    text-align: center;
+    padding: 10px;
+    list-style-type: none;
+}
+
+.main-nav__item__title {
+    text-decoration: none;
+    color: blue;
+    display: block;
+}
+
+.main-nav__item:hover {
+    background: #F1F1F1;
+    color: #333;
+    border-bottom: 3px solid lawngreen;
+}
+
+.main-nav__item__title:active {
+    color: orange;
+}
+
+.main-nav__item--active {
+    float: left;
+    text-align: center;
+    padding: 10px;
+    list-style-type: none;
+    background: #F1F1F1;
+    color: #333;
+    border-bottom: 3px solid lawngreen;
+}
+
+.main-nav__item--active__title {
+    color: orange;
+}
+
+
+/*end main-nav*/
+
+
+/*slider*/
+
+.slider {
+    border: 1px solid red;
+    background-color: grey;
+    position: relative;
+    padding: 0 90px;
+    margin: 10px;
+}
+
+.slider__prev {
+    float: left;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 50px;
+    text-decoration: none;
+    text-align: center;
+    color: white;
+    position: absolute;
+    top: calc(50% - 30px);
+    left: 50px;
+    font-weight: 100;
+}
+
+.slider__prev::before {
+    content: '\003C';
+}
+
+.slider__next {
+    content: '\0000';
+    float: right;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 50px;
+    text-decoration: none;
+    text-align: center;
+    color: white;
+    position: absolute;
+    top: calc(50% - 30px);
+    right: 50px;
+}
+
+.slider__next::before {
+    content: '\003E';
+}
+
+.slider__content {
+    height: 300px;
+    border: 1px solid red;
+}
+
+.slider__content__photo {
```

Comment:
I do not think, photo was part of slider. Please check and let me know if any question